src/test/resources/sya/allDetails.json

```diff
@@ -4,7 +4,7 @@
     "code": "PIP"
   },
   "mrn": {
-    "dwpIssuingOffice": "1",
+    "dwpIssuingOffice": "DWP PIP (1)",
```

Comment:
Aren't these sent from SYA as numbers? So not sure why this json has changed

Reply:
nope, they are sent from the SYA like that: DWP PIP (1). 
So this is making the json more realistic to Prod/Live data.

Reply:
ok

Reply:
What is the point of this class then? https://github.com/hmcts/sscs-tribunals-case-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/util/Norm.java#L18 
I thought it was sent as a number and then we change the string to match `DWP PIP (`